### ISSUE TITLE
JDAT-1485: Imviz region support: Add ellipse

### ISSUE THREAD

**Issue by pllim** [spacetelescope/jdaviz#635]
PO wants to draw ellipses, not just circles. Blocked by glue-viz/glue-astronomy#31 .

Rotating them is out of scope here and is part of #634 (JDAT-1484). Also see #518.

https://jira.stsci.edu/browse/JDAT-1485
